portfolio/src/main/webapp/index.html

```diff
@@ -214,6 +215,58 @@ <h3>My WCA Profile</h3>
                     <p id="favorite-quote"></p>
                     <button onclick="getQuote();">Click here to see some of my favorite quotes</button>
                 </div>
+                
+               
+                <div class="interactive-chart">
+                    <div class="title">
+                        <h3>Let's get to know better! What is your favorite programming language?</h3>
+                    </div>
+                    <div class="content">
+                        <div class="radios">
+                            <form action="/prog-lang" method="POST">
+                                
+                                <label for="radio_c++">
+                                    <input type="radio" name="language" id="radio_c++" value="C++">
+                                    C++
+                                </label>
+
+                                <label for="radio_python">
+                                    <input type="radio" name="language" id="radio_python" value="Python">
+                                    Python
+                                </label>
+                                
+                                <label for="radio_java">
+                                    <input type="radio" name="language" id="radio_java" value="Java">
+                                    Java
+                                </label>
+                                
+                                <label for="radio_c">
+                                    <input type="radio" name="language" id="radio_c" value="C">
+                                    C
+                                </label>
+                                
+                                <label for="radio_javascript">
+                                    <input type="radio" name="radios" id="radio_javascript" value="Javascript">
```

Comment:
Found a small bug :) name should be "language", otherwise it is not included in the same single-selection form. (I found it out trying to play around with your website)